backstop_data/engine_scripts/puppet/beforeTakingScreenshotHelper.js

```diff
@@ -0,0 +1,28 @@
+module.exports = async (page, { beforeTakingScreenshot }) => {
+  const allowedPageActions = {}
+
+  allowedPageActions.click = async selector =>
+    await page.click(selector);
+
+  allowedPageActions.hover = async selector =>
+    await page.hover(selector);
+
+  allowedPageActions.focus = async selector =>
+    await page.focus(selector);
+
+  allowedPageActions.type = async ({ into, text }) =>
```

Comment:
I noticed .type, .press and .waitFor async functions are not used but I am happy for them to remain here.